test/pflow_overlap_test.jl

```diff
@@ -0,0 +1,35 @@
+using CUDA
+using ExaPF
+using KernelAbstractions
+using Test
+
+import ExaPF: PowerSystem
+
+function foo(overlap=0)
+    #datafile = joinpath(dirname(@__FILE__), "data", "case9.m")
+    #datafile = joinpath(dirname(@__FILE__), "data", "case14.m")
+    datafile = joinpath(dirname(@__FILE__), "data", "case300.m")
+    #datafile = joinpath(dirname(@__FILE__), "data", "case9241pegase.m")
```

Comment:
This should use LazyArtifacts and the artifact maybe?

Reply:
Also just default case as a function argument.

Reply:
I think this script should be removed indeed